src/app/api/common/citizens/getCitizens.ts

```diff
@@ -15,7 +16,7 @@ type citizen = {
   voting_power?: Prisma.Decimal;
 };
 
-export async function getCitizens({
+async function getCitizens({
```

Comment:
the api.* prefix is just an import alias to prevent collisions with "use server" wrappers that are already defined. Here, I'm following the convention that
get**** -> raw/"unwrapped"/non-cached external db/resource/whatever access
fetch**** -> cache-wrapped

Reply:
there are plenty of duplicate import/re-export cases that are redundant (see src/app/admin/actions.ts) the fixing of which would probably obviate the need for the import alias but I didn't want to tangle with that ball of yarn on this PR